scenarios/basics/entity_resolution/SPECIFICATION.md

```diff
@@ -0,0 +1,289 @@
+An Entity Resolution matching workflow identifies and links records
+across datasets that represent the same real-world entity, such as
+customers or products. Using techniques like schema mapping,
+data profiling, and machine learning algorithms,
+it evaluates attributes like names or emails to detect duplicates
+or relationships, even with variations or inconsistencies.
+The workflow outputs consolidated, de-duplicated data, 
+
+
+Enter 'c' followed by <ENTER> to continue:
+c
+Continuing with the program...
+
+Workflow created successfully.
+The workflow ARN is: arn:aws:entityresolution:us-east-1:814548047983:matchingworkflow/MyMatchingWorkflow450
+
+Enter 'c' followed by <ENTER> to continue:
+c
+Continuing with the program...
+
+--------------------------------------------------------------------------------
+3. Start the matching job of the MyMatchingWorkflow450 workflow.
+
+Enter 'c' followed by <ENTER> to continue:
+c
+Continuing with the program...
+
+Job ID: ec2dbd1717624b2b806ed93a04c20049
+The matching job was successfully started.
+
+Enter 'c' followed by <ENTER> to continue:
+c
+Continuing with the program...
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+4. Get details for job ec2dbd1717624b2b806ed93a04c20049
+
+Enter 'c' followed by <ENTER> to continue:
+c
+Continuing with the program...
+
+Job status: QUEUED
+Job details: GetMatchingJobResponse(JobId=ec2dbd1717624b2b806ed93a04c20049, StartTime=2025-01-30T18:37:57.475Z, Status=QUEUED)
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+5. Get Schema Mapping.
+
+Enter 'c' followed by <ENTER> to continue:
+c
+Continuing with the program...
+
+Attribute Name: id, Attribute Type: UNIQUE_ID
+Attribute Name: name, Attribute Type: STRING
+Attribute Name: email, Attribute Type: STRING
+Schema mapping retrieval completed.
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+6. List Schema Mappings.
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+7. Tag the schema450resource.
+Tags can help you organize and categorize your Entity Resolution resources.
+You can also use them to scope user permissions by granting a user permission
+to access or change only resources with certain tag values.
+In Entity Resolution, SchemaMapping and MatchingWorkflow can be tagged. For this example,
+the SchemaMapping is tagged.
+
+Successfully tagged the resource.
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
```

Comment:
This output is missing the results section, and the delete resources section. It should be an exact copy of the output when the program is run.

Reply:
updated